peekaboo/sample.py

```diff
@@ -44,23 +45,6 @@
 logger = logging.getLogger(__name__)
 
 
-def make_sample(file, socket):
```

Comment:
@SebastianDeiss: Do you remember why you introduced this function? Seems a shame to me to wrap a nice class with a function...

Reply:
Before we had ``SocketServer`` we had one very big and ugly function, that was server and request handler. So, I introduced ``make_sample`` to make the server function more readable and I didn't change it after moving to ``SocketServer``.